README.md

```diff
@@ -1,70 +1,38 @@
-### Docker users, see [Moby and Docker](https://mobyproject.org/#moby-and-docker) to clarify the relationship between the projects
-
-### Docker maintainers and contributors, see [Transitioning to Moby](#transitioning-to-moby) for more details
-
 The Moby Project
 ================
 
 ![Moby Project logo](docs/static_files/moby-project-logo.png "The Moby Project")
 
-Moby is an open-source project created by Docker to advance the software containerization movement.
-It provides a “Lego set” of dozens of components, the framework for assembling them into custom container-based systems, and a place for all container enthusiasts to experiment and exchange ideas.
-
-# Moby
-
-## Overview
+Moby is an open-source project created by Docker to enable and accelerate software containerization.
```

Comment:
Is "Moby" (as a software name) also the successor of (the daemon side of) Docker Engine?

Reply:
Moby is the name of the whole project, which includes the moby github org, and other projects (linuxkit etc) that have their own orgs - everything mentioned on mobyproject.org. But I am also using it as a name for this repo, the Moby Project central repo, which is an integration point for a lot of pieces, including the current daemon. The actual binary naming is discussed in #34226

Reply:
@AkihiroSuda But to clearly answer you: no, moby ≠ engine, but moby > engine.

Reply:
@justincormack @tiborvass ok, thank you for explanation